Changefeed/ExampleEventCreator/ExampleEventCreator.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Azure.Storage.Blobs;
+using System;
+using System.IO;
+using System.Text;
+using System.Configuration;
+using System.Threading;
+
+namespace ExampleEventCreator
+{
+    class ExampleEventCreator
+    {   
+        static int count;
+        // Storage account connection string
+        static string connectionString = ConfigurationManager.AppSettings["connectionString"];
+        // Name of created container
+        static string containerName = "test-changefeed-container";
+
+        // Creates container and blobs within a storage account to populate changefeed and add example events
+        static void Main(string[] args)
+        {
+            count = 0;
+            
+            BlobServiceClient blobServiceClient = new BlobServiceClient(connectionString);
+            BlobContainerClient blobContainerClient = blobServiceClient.CreateBlobContainer(containerName);
+
+            Timer timer = null;
+            Console.WriteLine("Press Space to start the timer. Press Enter to stop the timer.");
+            while (true)
+            {
+                if(Console.ReadKey().Key == ConsoleKey.Spacebar)
+                {
+                    Console.WriteLine("Starting timer...");
+                    timer = new Timer(new TimerCallback(CreateEvents), null, 0, 1800000);
```

Comment:
Does this mean that every time the user presses the space bar they restart the timer / makes a new timer?
Shouldn't we wait for the user to press the space bar before starting the while loop.
Then for the while loop, instead of the condition being `true` should we have it be "Console.ReadKey().Key != ConsoleKey.Enter" ?